File: act3/People/Person.py
class Person():
	max_health=100
	health=100
	name=''
	food=0
	max_health=100
	def __init__(self,name):
		self.name=name
	def eat(self,food):
		self.health=self.health+food
		if (self.health>=max_health):
			self.health=max_health
		if (self.health<=0):
			self.health=0
	def is_alive(self):
		if(self.health > 0):
		    return True
		return False

File: act5/People/Person.php
<?php
require_once "interfaces/Human.php";

	abstract class Person implements Human
	{

		protected $_name;
		public $health = 100;
		public $max_health = 100;
		public function getName()
		{
			return $this->_name;
		}
		public function isalive()
		{
			if($this->health > 0)
			{
				return True;
			}
			else
			{
				return False;
			}
		}
		public function eat($food)
		{
			$this->health = $this->health + $food;
			if($this->health >= $max_health)
				$this->health = $max_health;
			if($this->health <= 0)
				$this->health=0;
		}
		
		public function move()
		{
		}

	}
